frontend/styles.css

```diff
@@ -56,7 +56,7 @@
     @apply hidden;
   }
   /* Generic link style */
-  a:not(.link-header), select, .link {
+  a:not(.link-header):not(.link-none), .link {
```

Comment:
@4down curious what the purpose of `.link-none` is?  Was something not achievable in core tailwind?  When you do this notice it adds extra size to the compiled css and it's not as easy to decipher.